content/v2.0-preview/_index.md

```diff
@@ -1,51 +1,53 @@
 ---
-title: "Overview"
+title: "Welcome"
 weight: -1
 cascade:
     version: "2.0-preview"
 ---
 
-{{< img src="/media/banner.png" alt="Crossplane Popsicle Truck" size="large" >}}
+Welcome to the Crossplane documentation. Crossplane is a control plane framework
+for platform engineering. 
 
-<br />
+Crossplane lets you build control planes to manage your cloud native software.
+It lets you design the APIs and abstractions that your users use to interact
+with your control planes.
 
-Crossplane is an open source Kubernetes extension that transforms your Kubernetes 
-cluster into a **universal control plane**. 
+Crossplane has a rich ecosystem of extensions that make building a control plane
+faster and easier. It's built on Kubernetes, so it works with all the Kubernetes
+tools you already use.
 
-Crossplane lets you manage anything, anywhere, all through standard Kubernetes
-APIs. Crossplane can even let you 
-[order a pizza](https://blog.crossplane.io/providers-101-ordering-pizza-with-kubernetes-and-crossplane/) 
-directly from Kubernetes. If it has an API, Crossplane can connect to it. 
+{{< hint "tip" >}}
+**A control plane is software that controls other software.**
 
-With Crossplane, platform teams can create new abstractions and custom
-APIs with the full power of Kubernetes policies, namespaces, role based access
-controls and more. Crossplane brings all your non-Kubernetes resources under
-one roof.
+Control planes expose an API. You use the API to tell the control plane what
+software it should configure and how - this is your _desired state_.
 
-Custom APIs, created by platform teams, allow security and compliance
-enforcement across resources or clouds, without exposing any complexity to the
-developers. A single API call can create multiple resources, in multiple clouds
-and use Kubernetes as the control plane for everything.
+A control plane can configure any cloud native software. It could deploy an app,
+create a load balancer, or create a GitHub repository.
```

Comment:
i think it'd be good to still mention an example of cloud infrastructure so people don't lose the fact that we still very much do infrastructure in addition to now being better suited for application level components :)

Reply:
I tried to do that by including the example of a load balancer. Is there a better example I could use? A more recognizable brand like RDS or ELB?

Reply:
i keep reading "load balancer" and thinking it's an in-cluster k8s component, even though it's not. I wonder if other folks make the same mental mistake. Regardless, I do think a more recognizable clear infrastructure piece could work better, such as a database.